spec/javascripts/social_networking/home-controller_spec.js

```diff
@@ -42,14 +43,15 @@ describe('HomeCtrl', function() {
     }
   });
 
-  beforeEach(inject(function($rootScope, $q, $controller, $filter, homeTool) {
+  beforeEach(inject(function($rootScope, $q, $controller, $filter, _homeTool_) {
+    homeTool = _homeTool_;
     scope = $rootScope;
     q = $q;
     controller = $controller('HomeCtrl', {
       OnYourMindResource: onYourMindResource,
       CommentResource: commentResource,
       LikeResource: likeResource,
-      homeTool: homeTool,
+      homeTool: _homeTool_,
```

Comment:
@eschlange in case @michaelwehrley doesn't see this tonight, here's the relevant documentation: https://docs.angularjs.org/api/ngMock/function/angular.mock.inject#resolving-references-underscore-wrapping-